src/three-components/Renderer.js

```diff
@@ -92,12 +92,20 @@ export default class Renderer extends EventDispatcher {
   }
 
   set exposure(exposure) {
+    if (!this.canRender) {
```

Comment:
Is this related to this PR?

Reply:
Yes, in Chrome 41 without WebGL this code path was being hit. We weren't seeing it before because without this snippet Chrome 41 wasn't even getting this far in our docs.

Reply:
Heh, good catch!